frontend/src/components/homepage/pageheader.tsx

```diff
@@ -0,0 +1,60 @@
+import {DropdownMenu,
+    DropdownMenuTrigger,
+    DropdownMenuContent,
+    DropdownMenuItem,
+    DropdownMenuSeparator,
+    } from "../ui/dropdown-menu"
+import { Button } from "../ui/button"
+
+function Pageheader() {
+    return( 
+    <div className="p-2">
+        <div className="flex md:hidden left-0">
+        <DropdownMenu>
+            <DropdownMenuTrigger className="w-full bg-transparent text-green-800">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" className="w-6 h-6">
+                <path d="M5.625 3.75a2.625 2.625 0 1 0 0 5.25h12.75a2.625 2.625 0 0 0 0-5.25H5.625ZM3.75 11.25a.75.75 0 0 0 0 1.5h16.5a.75.75 0 0 0 0-1.5H3.75ZM3 15.75a.75.75 0 0 1 .75-.75h16.5a.75.75 0 0 1 0 1.5H3.75a.75.75 0 0 1-.75-.75ZM3.75 18.75a.75.75 0 0 0 0 1.5h16.5a.75.75 0 0 0 0-1.5H3.75Z" />
+            </svg>
+            </DropdownMenuTrigger>
+            <DropdownMenuContent className="w-96 bg-emerald-800">
+                <DropdownMenuSeparator />
+                <DropdownMenuItem className="h-14">Home</DropdownMenuItem>
+                <DropdownMenuItem className="h-14">Discover Event</DropdownMenuItem>
+                <DropdownMenuItem className="h-14">Community</DropdownMenuItem>
+                <DropdownMenuItem className="h-14">About Us</DropdownMenuItem>
+                <DropdownMenuItem className="h-14">FAQs</DropdownMenuItem>
+                <DropdownMenuItem className="h-14">Log In/Sign Up</DropdownMenuItem>
+            </DropdownMenuContent>
+        </DropdownMenu>
+
+        </div>
+        <div className={`hidden md:flex flex-row mx-0$ {showFullWidthSearch ? "flex" : "hidden md:flex"}`}>
```

Comment:
Remove everything and replaced it to look like this

![Screenshot 2024-04-17 at 11 20 49 AM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/c446d54d-a265-49f1-a0c8-3ff2c6781284)